sage-ios/SAGE/SAGE/SplashView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SplashView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 4/21/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SplashView: UIView {
+    
+    var splashView = UIImageView()
+    var animated: Bool
+    let animationDuration = 1.7
```

Comment:
this needs to be a goddamn `TimeInterval`

Reply:
"i want to be a float" -animationDuration